concent_api/core/payments/backends/sci_backend.py

```diff
@@ -3,12 +3,12 @@
 from enum import Enum
 
 from typing import Callable
-
 from golem_sci import SmartContractsInterface
 from golem_sci.blockshelper import BlocksHelper
 from web3 import Web3
 
 from core.constants import ETHEREUM_ADDRESS_LENGTH
+from core.exceptions import BanksterTimestampError
```

Comment:
I've deleted that exception in my PR, because it wasn't use anywhere anymore, but I see it now here. I just want to point that out. It will cause some conflicts for me or You, depended on which PR is going to be merged first.